cordova/build.sh

```diff
@@ -93,6 +93,16 @@ if [ ! -d $PROJECT ]; then
     checkOK
   fi
 
+  if [ $CURRENT_OS == "IOS" ]; then
+    cordova plugin add phonegap-plugin-push --variable SENDER_ID="XXXXXXX"
```

Comment:
Sender id maps to the project number in the Google Developer Console. So, as we are using for only IOS application we can put in anything for this value.